src/web/src/alert_list.ecpp

```diff
@@ -126,8 +128,8 @@ bool database_ready;
         else
             if (!is_requeststate (state.c_str ())) {
                 log_error ("state = '%s' is not a valid alert state.", state.c_str ());
-                http_die ("request-param-bad", "state", std::string ("value '").append (state).append ("'").c_str (),
-                        std::string ("one of the following values [ ALL | ALL-ACTIVE | ACTIVE | ACK-WIP | ACK-IGNORE | ACK-PAUSE | ACK-SILENCE | RESOLVED ]").c_str ());
+                http_die ("request-param-bad", "state", TRANSLATE_ME (std::string ("value '")).append (state).append ("'").c_str (),
+                          (TRANSLATE_ME (std::string ("one of the following values")).append(std::string(" [ ALL | ALL-ACTIVE | ACTIVE | ACK-WIP | ACK-IGNORE | ACK-PAUSE | ACK-SILENCE | RESOLVED ]"))).c_str ());
```

Comment:
are you sure that this construct will stand the pace in the end?

Reply:
no